src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -47,50 +65,79 @@ trait WorkflowManager {
 /**
  * A WorkflowManager using WorkflowActors as the workflow handler
  */
-trait ActorWorkflowManager extends WorkflowManager {
-  override type Workflow = ActorRef // TODO: In a world where Akka Typed is no longer experimental switch to that
+
+object ActorWorkflowManager {
+  def props: Props = Props(classOf[ActorWorkflowManager])
+}
+
+class ActorWorkflowManager extends WorkflowManagerActor {
   override val backend = new LocalBackend
-  implicit def actorSystem: ActorSystem
+  implicit val timeout = Timeout(30.seconds)
 
-  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: WorkflowInputs): Try[Workflow] = {
+  override def generateWorkflow(id: WorkflowId, wdl: WdlSource, inputs: binding.WorkflowInputs): Try[Workflow] = {
     val binding = Try(WdlBinding.process(wdl))
-    binding map {x => actorSystem.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
+    binding map {x => context.actorOf(WorkflowActor.buildWorkflowActorProps(id, x, inputs))}
   }
 
-  override def workflowStatus(id: WorkflowId): Option[WorkflowState] = workflowById(id) map {x => workflowState(x)}
+  override def workflowOutputs(id: WorkflowId): Future[Option[binding.WorkflowOutputs]] = {
+    workflowById(id) match {
+      case Some(x) =>
+        val eventualSymbolStoreEntries = {x ? GetOutputs}.mapTo[Set[SymbolStoreEntry]]
+        symbolsToWorkflowOutputs(eventualSymbolStoreEntries) map {Option(_)}
+      case None => Future {None}
+    }
+  }
 
-  override def submitWorkflow(wdl: WdlSource, inputs: WorkflowInputs): Try[ManagedWorkflow] = {
-    val managedWorkflow = super.submitWorkflow(wdl, inputs)
-    managedWorkflow foreach {_.workflow ! Start(backend)}
-    managedWorkflow
+  override def submitWorkflow(wdl: WdlSource, inputs: binding.WorkflowInputs): Future[Try[WorkflowId]] = {
+    val eventualManagedWorkflow = addWorkflow(wdl, inputs)
```

Comment:
actually intellij suggests `eventualTriedManagedWorkflow` so I'll leave it up to you on either that and no type or `mw` plus type